dashboard/config/routes.rb

```diff
@@ -751,7 +755,9 @@
   get 'foorm/preview/:name', to: 'foorm_preview#name', constraints: {name: /.*/}
   get 'foorm/preview', to: 'foorm_preview#index'
 
-  get 'foorm/editor', to: 'foorm_editor#index', constraints: {name: /.*/}
+  get 'foorm/editor', to: 'foorm_form_editor#index'
```

Comment:
It might be nice to keep all the foorm routes together.  Any idea why these were separated before?

Reply:
mmm not 100% sure, the editor does feel like it's kind of separate from the other places we use foorm (there's sort of the "render a form for someone to fill out" part and "collect and do something with submissions to a given form" part). I agree, but there's kind of foorm stuff sprinkled throughout routes.rb and didn't want to bite off more cleanup than I could chew :).